inst/Upgrades.qml

```diff
@@ -0,0 +1,35 @@
+import QtQuick 		2.12
+import JASP.Module 	1.0
+
+Upgrades
+{
+
+	Upgrade
+	{
+		functionName:       "PrincipalComponentAnalysis"
+		fromVersion:		"0.16"
+		toVersion:			"0.17"
```

Comment:
We need to make sure that the versions here are correct. So If we merge it now then I think we need to have `fromVersion: "0.16"` and `toVersion: "0.16.1"`, if we wait for 0.17, we need `fromVersion: "0.16.1"` and `toVersion: "0.17"`, right @boutinb @JorisGoosen ?

Reply:
I think you are right.